showcase/app/templates/components/form/select.hbs

```diff
@@ -22,42 +22,77 @@
         </C.Options>
       </Hds::Form::Select::Base>
     </SF.Item>
+    <SF.Item @label="Multiple">
+      <Hds::Form::Select::Base multiple as |C|>
+        <C.Options>
+          <option selected>Lorem ipsum dolor</option>
+          <option>Sine qua non est</option>
+        </C.Options>
+      </Hds::Form::Select::Base>
+    </SF.Item>
+    <SF.Item @label="Groups">
+      <Hds::Form::Select::Base multiple size="8" as |C|>
```

Comment:
```suggestion
      <Hds::Form::Select::Base aria-label="groups select" multiple size="8" as |C|>
```
[Issue] - The lack of an `aria-label` here is causing the automated a11y checks to fail.